### DISCUSSION TITLE
render partial does not work

### DISCUSSION THREAD

**Discussion by randomsnowflake** [josefarias/hotwire_combobox#205] [Q&A]
I just installed the gem in my Rails 7 app and I am trying to render a combobox with a partial.

I use the following code, similar to the example in the docs:

```ruby
<%= combobox_tag "state", Entity.all.limit(10), render: { partial: "hotwiretest/item" } %>
```

However, it does not render the partial. Setting the partial name to a wrong name does not raise an exception, either.
It does render the combo box correctly though, using my getter function (and not the partial)

What am I doing wrong?

**Comment by vlado** [comment]
Try replacing `render` with `render_in`.

```
<%= combobox_tag "state", Entity.all.limit(10), render_in: { partial: "hotwiretest/item" } %>
```

**Reply by josefarias** [reply]
Great catch, thank you!
